servers/republik/modules/crowdfundings/lib/Mail.js

```diff
@@ -595,14 +605,23 @@ mail.prepareMembershipOwnerNotice = async ({ user, endDate, cancelUntilDate, tem
       { name: 'prolong_url',
         content: `${FRONTEND_BASE_URL}/angebote?package=PROLONG&token=${customPledgeToken}`
       },
+      { name: 'cancel_url',
+        content: `${FRONTEND_BASE_URL}/abgang?membershipId=${user.membershipIds.shift()}`
+      },
       { name: 'end_date',
         content: formattedEndDate
       },
+      { name: 'grace_end_date',
+        content: formattedGraceEndDate
+      },
       { name: 'days_left',
         content: daysLeft
       },
       { name: 'cancel_until_date',
         content: dateFormat(cancelUntilDate)
+      },
+      { name: 'sequence_number',
+        content: sequenceNumber
```

Comment:
why is it not the sequenceNumber of `user.memberships[0].sequenceNumber`? Why are the `membershipId` and `sequenceNumber` not dependent on each other?